### PR TITLE
Laboratory work #8, Aladyshkin Vasily - 20FPL1

### PR DESCRIPTION
**Pull request by froschalad**

### CODE REVIEW

File: lab_7_llm/tests/metric_check_test.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 """
```
Comment (SalnikovIgor) [inline lab_7_llm/tests/metric_check_test.py]: Checkout the files from 7 lab to the master

File: lab_8_llm/main.py [hunk 2]
```diff
@@ -150,6 +196,18 @@ def infer_sample(self, sample: tuple[str, ...]) -> str | None:
         Returns:
             str | None: A prediction
         """
+        tokens = self._tokenizer(sample[0], sample[1], max_length=120, padding=True,
+                                 return_tensors='pt', truncation=True)
+        with torch.no_grad():
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: Why use `torch.no_grad()`?

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
Add me (`SalnikovIgor`) as a contributor of your fork

File: lab_7_llm/tests/metric_check_test.py (hunk 1 above)
Comment (froschalad) [reply lab_7_llm/tests/metric_check_test.py]: All lab_7 files from the main branch are identical to files in my fork.

File: lab_8_llm/main.py (hunk 2 above)
Comment (froschalad) [reply lab_8_llm/main.py]: Done

**Pull request closed without merging**